tests/ibc-hooks/ibc_middleware_test.go

```diff
@@ -454,6 +456,11 @@ func (suite *HooksTestSuite) RelayPacket(packet channeltypes.Packet, direction D
 	ack, err := ibctesting.ParseAckFromEvents(receiveResult.GetEvents())
 	suite.Require().NoError(err)
 
+	if strings.Contains(string(ack), "error") {
+		errorCtx := gjson.Get(receiveResult.Log, "0.events.#(type==ibc-acknowledgement-error)#.attributes.#(key==error-context)#.value")
+		fmt.Println("ibc-ack-error:", errorCtx)
```

Comment:
Not sure if you meant to leave these printlns here (line 461, 513, and 654)

Reply:
I did! they only are displayed when there are errors, and I found myself having to put a breakpoint there just to find that string value when debugging. If they're annoying we can probably control it with a global var in that file `DEBUG_PRINTS = true` or something